core-js/src/main/javascript/cdf/components/ProtovisComponent.js

```diff
@@ -23,11 +23,11 @@ define([
       if(this.parameters == undefined) {
         this.parameters = [];
       }
-
+      
       this.renderChart();
     },
 
-    render: function(values) {
+    _renderOuter: function(values) {
```

Comment:
If you see my comment below, maybe we can just call `render` to `_renderOuter` and not need both. So this rename here would not be needed.